webapp/src/styles/_utils.scss

```diff
@@ -34,6 +50,19 @@
     display: flex;
 }
 
+.align-item-center {
```

Comment:
There is a util class available in mattermost for this.

Reply:
okay but this might be used at many places so let's not depend on MM utils classes as there might be then utils for other things as well which we don't know

Reply:
We can see all the utils in `mattermost-webapp`. And these are the things that are provided by `react-bootstrap` also, so we can depend on `react-bootstrap` if not on mm.

Reply:
I agree but as I mentioned we have already defined and used our utils at many places in the project so, it would be better to continue with that for now otherwise it will take time finding utils then using them here

Reply:
We can take the help of vscode's global-replace - `Ctrl + shift + H` to easily change it.